src/api/client/Polymesh.ts

```diff
@@ -151,8 +151,7 @@ export class Polymesh {
 
     const { metadata, noInitWarn, typesBundle } = polkadot ?? {};
 
-    // Defer `await` on any checks to minimize total startup time
```

Comment:
If we make polkadot api with a bad node URL it will keep on spewing out the "abnormal closure" errors.

In theory we could parallelize the middleware check here as well, but since it depends on `Context`, which depends on polkadotApi, there is a circular dependency issue in this instance with our current setup.